# Bioinformatics Unit – icipe Annual Training Schedule 2020
## FEB
### Data Management for Open and Reproducible Genomics Research Workshop (2 days)
This is a foundational training, which aims to introduce participants to considerations in planning a genomics project to ensure reproducibility and openness as vital principles for modern research.

**Audience:** Life science researchers involved or interested in genomics research, people involved with genomic data management, no prerequisite 
>
    • Introduction to bioinformatics (Data; Central dogma of Mol Bio, Online resources, NGS tech)
    • Organising a genomic project workspace (Data Management Plan)
    • Open and reproducible Bioinformatics research (Data tidiness, Frictionless Data tools)
    • Introduction to data backup and version control (Git and GitHub, Pipeline tools)

## MARCH
### Survery and Reasearch Data Management with REDCap (End of March)
We have the H3Africa meeting, where *icipe* will host the fellows training. We are planning for a REDCap training at the end of March. This will cover:
    - How to use REDCap for surveys
    - Setting up rubust surveys (We need to find a trainer on this)
    - Research data management


## APRIL - One Week
### R for reproducible research and statistical analysis
This can be a one-week training that targets data analysis using R studio, where we will also emphasise on project management. 

## JUNE - JULY
### EANBiT Residential Training Course (6 weeks)
This training aims at strengthening bioinformatics technical capacity and skills among the participants; preparing them to work as bioinformaticians and to become project ready so that they have the necessary skills to offer bioinformatics support in projects they are attached to.

**Audience:** Postgraduates ++ with familiarity to computing/programming and/or life science background
>
    • Version control and collaborative development (Git & Github & Slack etc.)
    • Advanced Scripting
    • Biological databases and API
    • Gene models and annotation
    • Reproducibility and package management: workflow languages (CWL, Snakemake, Conda)
    • Specialised databases (VectorBase, EupathDB) and APIs
    • Phylogenomics (Visualization and Annotation)
    • Metagenomics (16S and Whole-genome shotgun sequencing) 
    • Machine Learning and modelling for big data (fuzzy strings, HMM)

## AUGUST - OCTOBER
### H3ABioNet: Intermediate 16S rRNA Microbiome Training Course
The course aims to provide training to enable participants to gain the knowledge and skills to perform 16S rRNA microbiome data analyses using a variety of bioinformatics methods and tools.

**Audience:** Life science researchers involved or interested in genomics data analysis, no prerequisite 
>
    • Introduction to the Linux command line/intro to R
    • Introduction to the microbiome and study design – why 16S?
    • Sample collection, extraction and library prep for 16S NGS analyses
    • 16S rRNA gene amplicon sequencing bioinformatics pipeline: the theory
    • 16S analysis pipeline - QC, ASV picking, taxonomic classification and alignment metagenomeSeq (among others)
    • Downstream analysis in R - using the packages phyloseq, NMF, vegan,

## NOVEMBER
### A training on FAIR Data, focusing on metadata management
This two-day training will target anyone that handles data in any form in the institution. 

## DECEMBER
### Data Carpentry Genomics Workshop (4 days)
This workshop aims to teach data management and analysis for genomics research including best practices for the organisation of bioinformatics projects and data, use of command-line utilities, use of command-line tools to analyse sequence quality and perform variant calling, and connecting to and using cloud computing.

**Audience:** Life science researchers involved or interested in genomics data analysis, no prerequisite 
>
    • Project organisation and management
    • Introduction to the command line interface
    • Data wrangling and processing
    • Introduction to cloud computing
    • Introduction to R and RStudio for genomics
